<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenAvilogWin</class>
 <widget class="QWidget" name="GenAvilogWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>一键生成avilog</string>
  </property>
  <widget class="QLineEdit" name="Edit_CarPath">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_BigAvi">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>105</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>在线板</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_BigAviByCar">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>105</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>按料号存储</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>521</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Start">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CarPath">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Car路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Edit_AvilogPath">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_AvilogPath">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Avilog路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_OpenAvilog">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>···</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_OpenCarPath">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>···</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
